From 0b5223bd0b2544fa82710b739582214cec5422e2 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Fri, 29 Mar 2019 18:56:18 +0100
Subject: [PATCH] TRITON-1354 AdminUI should use bootstrap mode for moray -
 binder connections Reviewed by: Mike Gerdts <mike.gerdts@joyent.com> Reviewed
 by: Alex Wilson <alex.wilson@joyent.com> Approved by: Alex Wilson
 <alex.wilson@joyent.com>

---
 lib/sdc-clients.js              | 8 +-------
 package.json                    | 2 +-
 sapi_manifests/adminui/template | 9 +++------
 3 files changed, 5 insertions(+), 14 deletions(-)

diff --git a/lib/sdc-clients.js b/lib/sdc-clients.js
index c058af8c..e60ab676 100644
--- a/lib/sdc-clients.js
+++ b/lib/sdc-clients.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -103,12 +103,6 @@ module.exports.createClients = function (config) {
 
         var cfg = clone(opts.config);
         cfg.log = opts.log;
-        cfg.reconnect = true;
-        cfg.retry = {
-            retries: 999999,  // sdc-clients won't take Infinity
-            minTimeout: 1000,
-            maxTimeout: 16000
-        };
 
         var client = moray.createClient(cfg);
 
diff --git a/package.json b/package.json
index 92a6807a..e9e1767c 100644
--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "author": "Kevin Chan",
   "name": "adminui",
   "description": "Triton Operations Portal",
-  "version": "1.1.1",
+  "version": "1.1.2",
   "private": true,
   "repository": {
     "git": "git@git.github.com:joyent/sdc-adminui.git"
diff --git a/sapi_manifests/adminui/template b/sapi_manifests/adminui/template
index 74a02725..beae67ad 100644
--- a/sapi_manifests/adminui/template
+++ b/sapi_manifests/adminui/template
@@ -37,12 +37,9 @@
 		"{{datacenter_name}}": {
 			"rabbitmq": "{{{rabbitmq}}}",
 			"moray": {
-				"host": "{{{MORAY_SERVICE}}}",
-				"port": 2020,
-				"connectTimeout": 200,
-				"retry": {
-					"retries": 2,
-					"minTimeout": 500
+				"srvDomain": "{{{MORAY_SERVICE}}}",
+				"cueballOptions": {
+					"resolvers": [ "{{{BINDER_SERVICE}}}" ]
 				}
 			},
 			"amon": {
-- 
2.21.0

